components/eam/bld/namelist_files/use_cases/1850_eam_CMIP6.xml

```diff
@@ -22,6 +22,11 @@
 <prescribed_volcaero_filetype>VOLC_CMIP6					       </prescribed_volcaero_filetype>
 <prescribed_volcaero_type    >CYCLICAL						       </prescribed_volcaero_type    >
 <prescribed_volcaero_cycle_yr>1                                                        </prescribed_volcaero_cycle_yr>
+<!-- Removing prescribed volcanic forcing for CLDERA -->
```

Comment:
Why are these in the use_case file as opposed to just namelist defaults?

Reply:
Not sure. When I try to put them in the namelist_defaults, the use_cases overwrite them.

Reply:
Oh right, because the above lines override what's in defaults. That's annoying.